### PR TITLE
Amazon Inspector doc update

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to update the Amazon Inspector doc.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[AWS Inspector Doc Change ](https://sumologic.atlassian.net/browse/DOCS-349)

### CODE REVIEW

File: docs/integrations/amazon-aws/inspector.md [hunk 1]
```diff
@@ -20,10 +20,46 @@ Sumo Logic provides a serverless solution for creating a CloudWatch events rule
...
+
+This section demonstrates how to add a hosted Sumo Logic collector and AWS source, to collect events for the AWS Security Hub App.
+
+**Prerequisites**
+
+An HTTP source must be associated with a Sumo Logic Hosted collector. Before creating the source, identify the Sumo Logic Hosted collector you want to use, or create a new Hosted collector as described in the following task.
+
+To add a hosted collector and HTTP source:
+
+1. To create a new Sumo Logic Hosted collector perform the steps in [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/inspector.md:42]:
```suggestion
1. To create a new Sumo Logic Hosted collector perform the steps in [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/integrations/amazon-aws/inspector.md [hunk 2]
```diff
@@ -20,10 +20,46 @@ Sumo Logic provides a serverless solution for creating a CloudWatch events rule
...
+2. Add an [HTTP Source](/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source) to Sumo Logic, and in **Advanced Options for Logs**, under Timestamp Format, click **Specify a format** and enter the following:
+    * Specify **Format** as `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'`
+    * Specify **Timestamp locator** as `.*"UpdatedAt":"(.*)".*`
+    * Click **Add**.
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/inspector.md:46]:
```suggestion
    1. Specify **Format** as `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'`
    1. Specify **Timestamp locator** as `.*"UpdatedAt":"(.*)".*`
    1. Click **Add**.
```

**Review by jpipkin1 — APPROVED** [review]
Looks very good, @JV0812. I have a couple of suggestions.

File: docs/integrations/amazon-aws/inspector.md [hunk 3]
```diff
@@ -25,13 +25,13 @@ You can collect Security Hub logs using two methods:
 - [Method 1: Collect Security Hub logs using Sumo Logic HTTP endpoint](#method-1-collect-security-hub-logs-using-sumo-logic-http-endpoint)
 - [Method 2: Collect Security Hub logs using Amazon S3 source](#method-2-collect-security-hub-logs-using-amazon-s3-source)
 
-You do not have to manually create the AWS resources. Simply deploy the solution, as described in the [Step 2: Deploy an AWS Security Hub App collector](/docs/integrations/amazon-aws/inspector/#step-2-deploy-an-aws-security-hub-app-collector) for HTTP endpoint and [Step 2: Deploy an AWS Security Hub App collector](/docs/integrations/amazon-aws/inspector/#step-2-deploy-an-aws-security-hub-app-collector-1) for Amazon S3 source.
+You do not have to manually create the AWS resources. Simply deploy the solution, as described in the [Step 2: Deploy an AWS Security Hub app collector](/docs/integrations/amazon-aws/inspector/#step-2-deploy-an-aws-security-hub-app-collector) for HTTP endpoint and [Step 2: Deploy an AWS Security Hub app collector](/docs/integrations/amazon-aws/inspector/#step-2-deploy-an-aws-security-hub-app-collector-1) for Amazon S3 source.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/inspector.md:28]: It looks like that second URL doesn't exist

```suggestion
You do not have to manually create the AWS resources. Simply deploy the solution, as described in [Step 2: Deploy an AWS Security Hub app collector](/docs/integrations/amazon-aws/inspector/#step-2-deploy-an-aws-security-hub-app-collector).
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**